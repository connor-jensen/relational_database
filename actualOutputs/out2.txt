Schedule(D,'236')? Yes(1)
  D='M'