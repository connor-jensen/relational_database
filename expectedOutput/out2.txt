#checks if program will detect if query cannot be found

MT(X,'z')? Yes(2)
  X='x'
  X='y'
MT('y','z')? Yes(1)
MT(X,X)? Yes(1)
  X='y'
MT(X,Y)? Yes(3)
  X='x', B='z'
  X='y', B='y'
  X='y', B='z'
MT('z','z')? No